test/lib/graph/helpers/ui5Framework.js

```diff
@@ -1585,3 +1698,48 @@ test.serial("ProjectProcessor: Project missing in metadata", async (t) => {
 		message: "Failed to find library lib.a in dist packages metadata.json"
 	}, "Threw with expected error message");
 });
+
+test.serial("resolveSnapshotEndpointUrl", async (t) => {
```

Comment:
How is this test related to the ui5Framework helper? It doesn't use any of its APIs

Reply:
Inderectly thrugh the resolver. However, you're completely correct that it'd be better to move it to the Sapui5MavenSnapshotResolver test